app/assets/stylesheets/pages/home/_ukm.scss

```diff
@@ -0,0 +1,106 @@
+.ukm{
+	margin-top: 33px;
+
+	&__title-wrap{
+		text-align: center;
+		display: flex;
+		justify-content: center;
+		margin-bottom: 50px;
+	}
+
+	&__title{
+		font-size: 24px;
+		text-transform: uppercase;
+		width: 400px;
+		color: #484848;
+		line-height: 1.8;
+		margin-bottom: 0;
+	}
+
+	&__item-wrap{
+		display: flex;
+		justify-content: space-between;
+		flex-wrap: wrap;
+	}
+
+	&__item{
+		width: 305px;
+		box-sizing: border-box;
+		position: relative;
+		margin-top: 89px;
+
+		&:first-child{
```

Comment:
make the nth-child(2) and nth-child(3) styling become one with this

Reply:
okay, already combined nth-child 1 2 3 with nth-child(-n+3)